acceptance/tests/validate_vendored_ruby.rb

```diff
@@ -35,6 +35,7 @@ def setup_build_environment(agent)
     on(agent, "curl -O http://pl-build-tools.delivery.puppetlabs.net/solaris/11/sol-11-i386-compiler.tar.gz")
     on(agent, "gunzip -f sol-11-i386-compiler.tar.gz && tar -xf sol-11-i386-compiler.tar && rm -f sol-11-i386-compiler.tar")
     on(agent, "mv pl-build-tools/ /opt/")
+    on(agent, "ln -s i386/bin /opt/pl-build-tools/bin")
```

Comment:
This could be done also with, but not sure if it will break something else.
```
    on(agent, "mkdir -p /opt/pl-build-tools/bin/")
    on(agent, "mv pl-build-tools/i386/* /opt/pl-build-tools/")
    gem_install_sqlite3 = "export PATH=\"/opt/pl-build-tools/bin:/usr/sfw/bin:$PATH\" && #{gem_install_sqlite3}"
```